robottelo/performance/candlepin.py

```diff
@@ -121,7 +121,7 @@ def single_delete(cls, uuid, thread_id):
             LOGGER.error(
                 'Fail to delete {0} on thread-{1}!'.format(uuid, thread_id))
             LOGGER.error(response.content)
-            return
+            return 0
```

Comment:
I think is better to return None (`return`) if an error occurs. But if the errors happen and you really want do consider it as being 0 then ignore this comment.

By having None instead of 0 it can help you when generating the reports and if None is found you can mark as an error instead of a 0 result.